spec/js/shared/break_iterator.spec.js

```diff
@@ -27,10 +27,10 @@ describe("BreakIterator", function() {
     });
 
     it("does not split periods in the midst of other letters, eg. in a URL", function() {
-      var str = "Visit us. Go to http://translate.twitter.com.";
+      var str = "Visit us. Go to https://translate.twitter.com.";
```

Comment:
link resolved (lines 30 and 33)but redirects to note saying "Please note we are no longer accepting new Translation Center sign ups as of November 01, 2017."